src/modules/categories/components/CategoriesScreen.js

```diff
@@ -1,17 +1,17 @@
 import React, { Component } from 'react';
 import { FlatList, View } from 'react-native';
 import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
```

Comment:
`bindActionCreators` é mal, dificulta os testes :).

`The only use case for bindActionCreators is when you want to pass some action creators down to a component that isn't aware of Redux, and you don't want to pass dispatch or the Redux store to it.`

https://stackoverflow.com/questions/41754489/when-would-bindactioncreators-be-used-in-react-redux
http://redux.js.org/docs/api/bindActionCreators.html#bindactioncreatorsactioncreators-dispatch